pkg/store/kotsstore/kots_store.go

```diff
@@ -221,6 +221,10 @@ func canIgnoreEtcdError(err error) bool {
 		return true
 	}
 
+	if strings.Contains(err.Error(), "EOF") {
```

Comment:
@divolgin can we use `errors.Is` instead?

https://golang.org/pkg/errors/#example_Is

Reply:
Would love to, but I don't know which error it is without a way to reproduce it.  Also, this covers any EOF error.